core/spec/models/spree/product_spec.rb

```diff
@@ -640,4 +640,15 @@ class Extension < Spree::Base
       expect(result).to eq([product_1, product_2])
     end
   end
+
+  describe ".ransackable_scopes" do
+    subject { described_class.ransack(available: true).result }
```

Comment:
Nit, but the subject (available) doesn't correspond to what `describe` says is testing (generic).

Reply:
Do you have a suggestion @waiting-for-dev? What about `ransack scopes`? and then move the "it includes the available scope" to a context?

Reply:
Yes, that makes sense